.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -49,6 +49,24 @@ for test in ${CHECK_PACKAGES_TESTS[@]}; do
     fi
 done
 
+pushd test/packages/false_positives > /dev/null
+for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
+    package_name=$(basename ${package})
+    echo "      - label: \":go: Running integration test (false positive): ${package_name}\""
+    echo "        key: \"integration-false_positives-${package_name}\""
+    echo "        command: ./.buildkite/scripts/integration_tests.sh -t test-check-packages-false-positives -p ${package_name}"
+    echo "        env:"
+    echo "          UPLOAD_SAFE_LOGS: 1"
+    echo "        agents:"
+    echo "          provider: \"gcp\""
+    echo "        artifact_paths:"
+    echo "          - build/test-results/*.xml"
+    echo "          - build/elastic-stack-dump/check-*/logs/*.log"
+    echo "          - build/elastic-stack-dump/check-*/logs/fleet-server-internal/**/*"
```

Comment:
According to this thread https://github.com/elastic/elastic-package/pull/1333/files#r1250786653, let's just remove these two from `artifact_paths` and just keep the result xml files, as it happens for `parallel` test packages:
 ```suggestion
```